tools/osgicheck-maven-plugin/src/main/java/org/apache/felix/maven/osgicheck/impl/CheckContext.java

```diff
@@ -45,4 +48,20 @@ public interface CheckContext {
      * @param message The message.
      */
     void reportError(String message);
+
+    /**
+     * This method is invoked by a {@link Check} to report
+     * an exception.
+     * @param message The message.
+     */
+    default void reportError(String message, Throwable cause) {
```

Comment:
This does not compile:
```
[INFO] -------------------------------------------------------------
[ERROR] COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
[ERROR] /Users/gnodet/work/git/felix-dev/tools/osgicheck-maven-plugin/src/main/java/org/apache/felix/maven/osgicheck/impl/CheckContext.java:[57,5] default methods are not supported in -source 1.7
  (use -source 8 or higher to enable default methods)
[INFO] 1 error
[INFO] -------------------------------------------------------------
```

Reply:
Should be fixed now, I increased the bytecode level to Java 8.